bin/internal/shared.bat

```diff
@@ -155,7 +154,7 @@ GOTO :after_subroutine
       SET /A remaining_tries=%total_tries%-1
       :retry_pub_upgrade
         ECHO Running pub upgrade... 1>&2
-        "%dart%" __deprecated_pub upgrade "%VERBOSITY%" --no-precompile
+        "%dart%" --no-analytics pub upgrade
```

Comment:
is the `--no-precompile` flag no longer needed?

Reply:
No - it is the default in `dart pub get now` (for some years).